### DISCUSSION TITLE
Add additional zoom-out factor for maps exceeding original size limit

### DISCUSSION THREAD

**Discussion by tellovishous** [OpenRCT2/OpenRCT2#16980] [Feature requests]
Since the new save format allows map sizes about 4 times bigger than the original limit, I think we could also add an extra step to the zoom out function on bigger maps to account for this. Maps at the new limit max out at 999 squares (compared to the original limit of 254), and so maps beyond the original limit can totally fill most screens at the current max view, so another step that lets huge maps be viewed all at once (or at least show more at once) would be good to have.

**Comment by Broxzier** [comment]
With an additional zoom level, you'd be looking at a couple pixels per tile element. You're better of reducing your window scale factor instead, which you can bind to hotkeys and mouse buttons for convenience.

**Reply by ocalhoun6** [reply]
>With an additional zoom level, you'd be looking at a couple pixels per tile element.

I think that's acceptable. Think of it like a fullscreen version of the mini-map.

**Comment by tellovishous** [comment]
Okay sure, maybe you wouldn't see much in the way of detail at another zoom level, but if nothing else it could at least be useful for navigation, so one could zoom out to a wider view on giant maps, and then zoom in to particular section on the other side of it and thus could be done a little easier & quicker that way compared to combing around as much within the view. At least that's how I see it.
